app/controllers/streaks_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class StreaksController < ApplicationController
+  before_action :set_user, only: %i[show]
+
+  def show
+    render json: {current_streak: @user.current_streak}
```

Comment:
@castrolem should we add this to the subscription instead? I don't see a case where we need to query just the streak

Reply:
I thought it could be a bit of too much data to be available. What's your opinion?